analyses/focal-cn-file-preparation/run-prepare-cn.sh

```diff
@@ -19,92 +19,80 @@ script_directory="$(perl -e 'use File::Basename;
   print dirname(abs_path(@ARGV[0]));' -- "$0")"
 cd "$script_directory" || exit
 
+
+GENCODE="ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.annotation.gtf.gz"
+if [ ! -e ${GENCODE##*/} ]
+then
+  echo "Downloading ${GENCODE##*/}"
+  curl -O $GENCODE
+fi
+
 scratch_dir=../../scratch
-data_dir=../../data
+data_dir=../../data/
 results_dir=../../analyses/focal-cn-file-preparation/results
 histologies_file=${data_dir}/histologies.tsv
-gtf_file=${data_dir}/gencode.v27.primary_assembly.annotation.gtf.gz
+#gtf_file=${data_dir}/gencode.v38.primary_assembly.annotation.gtf.gz
+gtf_file=gencode.v38.annotation.gtf.gz
```

Comment:
oh - also, did not catch it before, but we want the primary_assembly gencode.

Reply:
@runjin326 Yes, thanks for bringing this up. I did notice this is not the primary_assembly file when I downloaded it, but I was unable to find a different link for v38 This was the only link I found for v38 annotation. Would you mind looking it up and sharing it with me please?